src/__main__.py

```diff
@@ -15,9 +15,11 @@
 import asyncio
 import logging
 
-from config import config
 # noinspection PyUnresolvedReferences
+from src import commands  # pylint: disable=unused-import
```

Comment:
[Question] If this import truly is unused, why suppress the linting error?

Reply:
its not unused. 

What happens here is `src.commands` package simply *needs to be loaded*, but its contents are not *directly* used.

This package contains our user-facing Interface, the commands system. Commands are registered at load time, but that **requires them to be loaded**

This seems like the best place to do this, is at the top of main as part of pre-initialization. 

The import needs to exist, even if its not used by this module directly.